src/sentry/static/sentry/app/components/group/suggestedOwnerHovercard.jsx

```diff
@@ -24,10 +26,25 @@ const SuggestedOwnerHovercard = ({actor, commits, rules, ...props}) => (
         {actor.id === undefined && (
           <EmailAlert icon="icon-warning-sm" type="warning">
             {tct(
-              'The email [actorEmail]  has no associated Sentry account. Make sure to link alternative emails in [accountSettings:Account Settings].',
+              'The email [actorEmail] has no associated Sentry account. You can [inviteUser:invite this user] to your organization or link additional emails in [accountSettings:account settings].',
```

Comment:
Is there a privacy concern here? i.e. is it possible to leak potential information about who our clients are?